content/id/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Mengatur Master Kubernetes dengan Ketersediaan Tinggi (High-Availability)
+content_type: task
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.5" state="alpha" >}}
+
+Kamu dapat mereplikasi master Kubernetes dalam skrip `kube-up` atau `kube-down` untuk Google Compute Engine (GCE).
```

Comment:
Let's replace all master with _control plane_, this comment applies to all "master" words in this page
```suggestion
Kamu dapat mereplikasi _control plane_ Kubernetes dalam skrip `kube-up` atau `kube-down` untuk Google Compute Engine (GCE).
```

Reply:
agree!